### PR TITLE
Stories/more rubocop

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/filters/month_and_organization.rb [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
+    attribute :org, Integer, default: ->(s, _) { s.default_org }
     attribute :month, Integer, default: Date.current.month
     attribute :year, Integer, default: Date.current.year
     attribute :user, User, default: nil
 
     validates :org, presence: true
 
     def months
-      @months = %w( January February March April May June July August September October November December ).each_with_index.to_a.map{ |m,i| [ m, i + 1 ] }
+      @months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'].each_with_index.to_a.map { |m, i| [m, i + 1] }
```
Comment (awisema) [inline app/models/filters/month_and_organization.rb]: Use `Date::MONTHNAMES`?

File: app/models/grda_warehouse/client_notes/alert.rb [hunk 2]
```diff
@@ -13,15 +13,13 @@ def self.type_name
     end
 
     # anyone who can see this client
-    scope :visible_by, -> (user, client) do
+    scope :visible_by, ->(user, client) do # rubocop:disable Lint/UnusedBlockArgument
```
Comment (awisema) [inline app/models/grda_warehouse/client_notes/alert.rb]: Is the disable any different than renaming the parameter `_client`? If not, I think I'd prefer the name change.

File: app/models/grda_warehouse/client_notes/service_note.rb [hunk 3]
```diff
@@ -11,9 +11,9 @@ def self.type_name
     end
 
     # anyone who can see the client and project
-    scope :visible_by, -> (user, client) do
+    scope :visible_by, ->(user, client) do # rubocop:disable Lint/UnusedBlockArgument
```
Comment (awisema) [inline app/models/grda_warehouse/client_notes/service_note.rb]: `_client`?

**Review by awisema — CHANGES_REQUESTED** [review]

File: .rubocop.yml [hunk 4]
```diff
@@ -145,6 +153,9 @@ Layout/MultilineOperationIndentation:
 Style/AccessModifierDeclarations:
   Enabled: false
 
+Style/RedundantReturn:
```
Comment (awisema) [inline .rubocop.yml:156]: Does this suppress the complaint for `return x` at the end of a method? I thought we had decided that was a good convention?

File: app/models/filters/month_and_organization.rb (hunk 1 above)
Comment (eanders) [reply app/models/filters/month_and_organization.rb]: I was wondering if you'd catch that. 😉

File: .rubocop.yml (hunk 4 above)
Comment (eanders) [reply .rubocop.yml:156]: It does.  We have enough legacy code where that was a pain to cleanup or disable, and it complains when the return is in an if statement in the middle of the method where the return isn't obvious.  I agree, we should not add more of them, but there were a few situations that I felt warranted the clarity it provides.

**Pull request merged**